x-pack/plugins/observability_solution/infra/public/plugin.ts

```diff
@@ -425,7 +431,8 @@ const getLogsExplorerAccessibility$ = (application: CoreStart['application']) =>
           AppStatus.accessible &&
         capabilities.discover?.show &&
         capabilities.fleet?.read
```

Comment:
What do you think about converting these to a boolean early so the return type is nicer?

```suggestion
        !!capabilities.discover?.show &&
        !!capabilities.fleet?.read
```